lib/global_uid.rb

```diff
@@ -27,3 +27,8 @@ class TimeoutException < Exception ; end
 if ActiveRecord::VERSION::MAJOR >= 4
   ActiveRecord::SchemaDumper.send(:prepend, GlobalUid::SchemaDumperExtension)
 end
+
+# Make sure that GlobalUID is disabled for ActiveRecord's Internal Metadata table
+if ActiveRecord::VERSION::MAJOR >= 5
+  ActiveRecord::InternalMetadata.global_uid_disabled
```

Comment:
You mean as a replacement for the ActiveRecord version or on top? I'm fine with the first solution but the second one would be unnecessary since it's defined in Rails 5 😄